refinery/workflow_manager/views.py

```diff
@@ -27,10 +27,10 @@ def import_workflows(request):
         logger.debug("Engine: " + engine.name + " - " +
                      str(new_workflow_count) + ' workflows after.')
         workflows += new_workflow_count
-    return HttpResponse(str(workflows) + ' workflows imported.')
+    return HttpResponse(str(workflows) + ' workflows imported.', )
```

Comment:
I only know about trailing commas when you want a 1-element tuple: Not sure what's going on here?

Reply:
That's a typo. :)